### ISSUE TITLE
Failure to start on Mac BigSur with Waiting for Kubernetes API (but works with painful workaround)

### ISSUE THREAD

**Issue by radder5** [rancher-sandbox/rancher-desktop#1336]
### Rancher Desktop Version
0.7.1

### What operating system are you using?
macOS

### Operating System / Build Version
macOS BigSur 11.6.2

### What CPU architecture are you using?
Intel

```
system_profiler SPNetworkDataType -json | jq -r '.SPNetworkDataType[] | select(.interface) | @text "\(._name):\(.interface)"'
Apple USB Ethernet Adapter:en7
Wi-Fi:en0
Bluetooth PAN:en6
Thunderbolt Bridge:bridge0
```
I only have Wifi connected.

### Expected behaviour
Rancher-desktop starts successfully

### Actual behaviour
I check the k3s.log and got invalid node ip, unable to get global unicast ip from interface name: can't find ip for interface rd0.
Rancher-desktop fails to start
Stuck on Waiting for Kubernetes API.... (message in UI)

Following suggestions from https://github.com/rancher-sandbox/rancher-desktop/issues/1129#issuecomment-997401259 I have changed the rd0 to en0 in lima.yaml which doesn't work exactly.
However, if I monitor the lima.yaml file whilst the rancher-desktop is starting, during initial startup where the UI message is "Starting virtual machine" at this point or before, the change in lima.yaml is reverted back to rd0. If I quickly change that again to en0 during this startup process and re-save, then rancher-deskop will start successfully.

If I stop rancher-desktop after this workaround and start again, it fails to start again as above. I can only restart if I carry out the edit of lima.yaml during startup.

**Comment by jandubois** [comment]
Please upgrade to the 1.0.0 release; this problem should be fixed. Re-open if it still happens to you afterwards!

**Comment by radder5** [comment]
> Please upgrade to the 1.0.0 release; this problem should be fixed. Re-open if it still happens to you afterwards!

It is indeed fixed with 1.0.0 - fantastic and thanks
